src/commands/statusline.ts

```
@@ -140,17 +152,40 @@ export const statuslineCommand = define({
+								type BurnStatus = 'normal' | 'moderate' | 'high';
+
+								const burnStatus: BurnStatus = burnRate.tokensPerMinuteForIndicator < 2000
+									? 'normal'
 									: burnRate.tokensPerMinuteForIndicator < 5000
-										? pc.yellow(costPerHourStr) // Moderate
-										: pc.red(costPerHourStr); // High
+										? 'moderate'
+										: 'high';
+
+								const burnStatusMappings: Record<BurnStatus, { emoji: string; textValue: string; coloredString: Formatter }> = {
+									normal: { emoji: '🟢', textValue: 'Normal', coloredString: pc.green },
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)